### PR TITLE
complete

### PR DESCRIPTION
**Pull request by Alexithemia**

### CODE REVIEW

**Review by jaywon — CHANGES_REQUESTED** [review]
Your solution is close but would not catch all cases. See if you can find out why! 🤔

File: multiples.js [hunk 1]
```diff
@@ -3,10 +3,13 @@
  * @param  {Number} n Number that is to be used for calculating multiples up to that Number
  * @return {Number}
  */
-exports.sumOfAMultiple = function( n ) {
+exports.sumOfAMultiple = function (n) {
   var sum = 0;
 
-  // do your work here
-
+  for (let i = 1; i < n; i++) {
```
Comment (jaywon) [inline multiples.js:9]: Your solution won't catch all cases, see if you can find out why 🤔

**Review by jaywon — CHANGES_REQUESTED** [review]
Plz see comment.

**Review by taesup — APPROVED** [review]
LGTM! Closing PR.

**Pull request closed without merging**